libs/libc/machine/risc-v/common/arch_elf.c

```diff
@@ -399,6 +400,29 @@ int up_relocateadd(FAR const Elf_Rela *rel, FAR const Elf_Sym *sym,
         }
         break;
 
+      case R_RISCV_JAL:
+        {
+          binfo("%s at %08" PRIxPTR " [%08" PRIx32 "] "
```

Comment:
why this is `%08`? maybe we can `PRIxREG` instead? Or only 32-bt address space is considered as valid?